docs/user-guide/load-balancer.md

```diff
@@ -71,3 +77,61 @@ information through `kubectl`:
     No events.
 
 The IP address is listed next to `LoadBalancer Ingress`.
+
+## Loss of client source IP for external traffic
+
+Due to the implementation of this feature, the source IP for sessions as seen in the target container will *not be the original source IP* of the client. This is the default behavior as of Kubernetes v1.4. However, starting in v1.4, an optional alpha feature has been added
+that will preserve the client Source IP for GCE/GKE environments. This feature will be phased in for other cloud providers in subsequent releases.
+
+## Annotation to modify the LoadBalancer behavior for preservation of Source IP
+In 1.4, an Alpha feature has been added that changes the behavior of the external LoadBalancer feature.
+
+This feature can be activated by adding the alpha annotation to the metadata section of the Service Configuration file.
+
+    {
+      "kind": "Service",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "example-service"
+        "annotations": {
+            "service.alpha.kubernetes.io/external-traffic": "OnlyLocal"
+        }
+      },
+      "spec": {
+        "ports": [{
+          "port": 8765,
+          "targetPort": 9376
+        }],
+        "selector": {
+          "app": "example"
+        },
+        "type": "LoadBalancer"
+      }
+    }
+
```

Comment:
you should explain what this does to achieve the result, including only routing locally, the fact that nodeports are not handled yet, and the basic healthchecks stuff